test/Manager-test.js

```diff
@@ -42,23 +50,44 @@ describe('Manager', () => {
 
   it('should be a function', () => {
     expect(Manager).to.be.a('function')
-  }),
+  })
   it('should be an instance of Manager', () => {
     expect(lucyPhurr).to.be.an.instanceof(Manager)
   })
   it('should sort bookings into todaysBookings and laterBookings', () => {
-    expect(lucyPhurr.getBookingStatus(bookingRepo)).to.equal(undefined)
-  }),
+    createBookingRepo()
+    expect(lucyPhurr.getBookingStatus(bookingRepo)).to.deep.equal([
+      bookingRepo[0],
+      bookingRepo[1],
+      bookingRepo[3],
+      bookingRepo[5]
+    ])
+  })
   it('should get booked rooms', () => {
-  expect(lucyPhurr.getBookedRooms(bookingRepo)).to.equal(undefined)
+    createBookingRepo()
+    expect(lucyPhurr.getBookedRooms(bookingRepo)).to.deep.equal([
+      roomRepo[0],
+      roomRepo[1],
+      roomRepo[6],
+      roomRepo[3]
+    ])
   })
   it('should get vacant rooms', () => {
-  expect(lucyPhurr.getVacantRooms(bookingRepo)).to.equal(undefined)
+    createBookingRepo();
+    expect(lucyPhurr.getVacantRooms(roomRepo, bookingRepo)).to.deep.equal([
+      roomRepo[2],
+      roomRepo[4],
+      roomRepo[5]
+    ])
   })
   it('should get the percentage of available rooms', () => {
-    expect(lucyPhurr.getPercentageOfRoomsAvailable()).to.equal(100)
+    createBookingRepo()
+    // let bookedRooms = lucyPhurr.getBookedRooms(bookingRepo)
+    // let getVacantRooms = lucyPhurr.getVacantRooms(roomRepo, bookingRepo)
+    expect(lucyPhurr.getPercentageOfRoomsAvailable(roomRepo, bookingRepo)).to.equal(43)
   })
-  it('should get the total revenue for today.', () => {
-    expect(lucyPhurr.getTotalRevenue()).to.equal(0)
+  it.only('should get the total revenue for today.', () => {
```

Comment:
Make sure to remove this it.only before proceeding with any more testing of this class! That can really mess things up of it is forgotten about!